pkg/results/result.go

```diff
@@ -166,31 +249,35 @@ type MergeableAttributes struct {
 }
 
 // RowsMap is an aggregated representation of a Rows list
-type RowsMap map[MergeableAttributes]*types.Counters
+type RowsMap map[MergeableAttributes]types.Counters
```

Comment:
OK, just did a quick check on that. On "normal" queries there doesn't seem to be any significant difference. But: Some change in this PR (don't know if it's this or some other) breaks raw queries completely. Running one on current develop (limiting to a reasonable amount of rows) works, on this branch it kills my machine no matter how low I set `n`. It also doesn't return the same results:
```
/usr/bin/time -v ../goQuery/goQuery -d /tmp/db_out2  -i eth0 -f "2010-01-01T00:00:00Z" -n 100 raw
Command terminated by signal 9
...
	Maximum resident set size (kbytes): 8095244
```

If I limit the time range (so it doesn't blow up in my face) I can at least compare properly:

**This branch:**
```
/usr/bin/time -v ./goQuery -d /tmp/db_out2  -i eth0 -f "2020-01-01T00:00:00Z" -n 10 raw
...
              Totals:                                                                                                                                                      1.29 G                      1.16 TB  
Timespan / Interface : [2020-01-01 00:59:48, 2020-04-10 10:18:12] / eth0
Sorted by            : first packet time
Query stats          : displayed top 10 hits out of 2.89 M in 4.2s
```

**Reference (develop):**
```
/usr/bin/time -v ./goQuery -d /tmp/db_out2 -i eth0 -f 2020-01-01T00:00:00Z -n 10 raw
...
                                 Totals:                                                                    1.42 G                    1.30 TB  
Timespan / Interface : [2020-01-01 00:59:48, 2020-04-10 10:18:12] / eth0
Sorted by            : accumulated data volume (sent and received)
Query stats          : displayed top 10 hits out of 116.38 k in 245ms
```

Note also the much faster response time of the "old" code. Something clearly is not right when it comes to raw queries.

Reply:
> > Obviously no guarantees to safety of the review (my head's spinning after going through this behemoth of a PR dizzy ).
> > Change requested because of the raw query issue (not sure if others are affected, too). I'll try to help investigate as much as I can.
> 
> Thanks for the initial review and the suggestions.
> 
> I'll also prioritize looking at the raw queries. Unfortunately never ran them during testing see_no_evil.

Sounds good. Please check out my latest comment though, I don't want to send you on a wild goose chase for nothing. I _think_ this is a weird mix of a regression that we've had for some (unknown) while on `develop` and _maybe_ something in this PR. What is still odd is the fact that the overall numbers in the example above don't add up, although clearly the same range was used (but that could be a side effect of some aggregation going on despite the fact that `raw` was specified)...

Reply:
Jo, thanks a lot for cross-checking this! You're right, this is exactly the reason for #75 . I'll increase priority on that one.